word_guess.rb

```diff
@@ -0,0 +1,325 @@
+require "colorize"
+class GamePlay
```

Comment:
I like the way you've separated things out into their different purposes. It seems like the ASCII art functionality (for each guess) could have also been separated out into a separate class or method.